src/pages/reference/compare-to-heroku.md

```diff
@@ -0,0 +1,93 @@
+
+## Differences
+
+### Builds
+
+Part of the limitations faced by the Heroku team during the early 2010s was the question of how to best ship code.
+
+At the time, the idea of unikernels and Docker didn't take hold until much later. As a result: their build solution, Buildpacks, used mostly complex Bash scripts to introspect into code to build your app.
+
+In the current era of software development, we feel that in some respects, knowing Docker to know how to ship your app is too much boilerplate for the average developer. But, we still want to offer the developer all the benefits of reproducibility, speed, and ease of use when deploying images, hence why we built our builder called Nixpacks.
+It's open source (https://github.com/railwayapp/nixpacks), and it's also swift.
+We detect more languages, builds are faster with the help of intelligent caching, and lastly, builds are reproducible. (A common frustration of Buildpacks.)
+Those moving from Heroku to Railway will find that we support CNCF Buildpacks just like Heroku does if you want to keep using their build solution.
+
+### Product
+
+We offer a single pane of glass for evolving your infrastructure over time. We are proud of the interfaces we provide, and our users report this is one of their favorite platform aspects.
+
+It's possible to go from zero → 3 services in <30 seconds. Adding new services trivially and meshing them together means you don't need to compromise on architecture because of your infrastructure.
+
+### Pricing
+
+Pay only for what you use—no need to size dynos.
+
+Containers scale vertically up to 8 GB/CPU Cores (on DevPlan) and 32 GB/CPU Cores (on Teams Plans), and you only pay per minute.
+
+Did traffic spike for an hour? Only pay for that hour, and no manual scaling intervention is required.
+
+For those who prefer to pay upfront. We offer credit-based Developer plans who wish to load their account with credits.
+
+### Develop and Deploy
+
+You can use the CLI to run every service locally, hooking in with your cloud infrastructure. Whenever you use
+e.g. `railway run yarn start` in a project with Postgres will automagically inject things like `DATABASE_URL` etc.
+
+If your changes look good, you can even run `railway up` from the terminal to trigger a new deployment on demand.
+
+### Lifecycle Management
+
+We offer guardrails to prevent new deployments from knocking your app offline, such as [Healthchecks](/diagnose/healthchecks). For example, by providing an endpoint for Railway to hit, we will make a series of requests, checking for a 200 OK before we switch deploys to the most recent one.
+
+Want to revert a change? Because we image all builds, we allow you to [rollback](/deploy/deployments#rollback) to an earlier working deploy in just one click.
```

Comment:
```suggestion
Want to revert a change? Because we image all builds, you can [rollback](/deploy/deployments#rollback) to an earlier working deploy in just one click.
```